src/twisted/internet/iocpreactor/tcp.py

```diff
@@ -141,17 +140,13 @@ def logPrefix(self):
         return self.logstr
 
     def getTcpNoDelay(self):
-        return operator.truth(
-            self.socket.getsockopt(socket.IPPROTO_TCP, socket.TCP_NODELAY)
-        )
+        return bool(self.socket.getsockopt(socket.IPPROTO_TCP, socket.TCP_NODELAY))
 
     def setTcpNoDelay(self, enabled):
         self.socket.setsockopt(socket.IPPROTO_TCP, socket.TCP_NODELAY, enabled)
 
     def getTcpKeepAlive(self):
-        return operator.truth(
-            self.socket.getsockopt(socket.SOL_SOCKET, socket.SO_KEEPALIVE)
-        )
+        return bool(self.socket.getsockopt(socket.SOL_SOCKET, socket.SO_KEEPALIVE))
```

Comment:
Replacing `operator.truth()` with `bool()` is fine.

Reply:
Right. `operator.truth` is documented as being the same as `bool`.